donut/modules/flights/templates/settings.html

```diff
@@ -0,0 +1,27 @@
+{% extends "layout.html" %}
+{% block page %}
+<div class="container theme-showcase" role="main">
+<div class="jumbotron">
+	<div class="col-sm-1">
+		    <a href="{{url_for('flights.flights')}}" class = "btn btn-primary back"> &lt;</a>
+	</div>
+	<div class="col-md-4">
+		<h2>Airport Carpool Settings</h2>
+		<form method="POST" action="{{ url_for('flights.update') }}">
+                	<label for="visible">Sheets are visible</label>
+			<input type="checkbox" id="visible" name="visible" {% if visible %} checked {% endif %}>
+			<br>
```

Comment:
I think you can wrap inputs in `<div class='form-group'>` so they will be spaced out appropriately